Lib/_markupbase.py

```diff
@@ -276,13 +276,14 @@ def _parse_doctype_attlist(self, i, declstartpos):
                 return -1
             if c == "(":
                 # an enumerated type; look for ')'
-                if ")" in rawdata[j:]:
-                    j = rawdata.find(")", j) + 1
+                _temp = rawdata.find(")", j)
```

Comment:
Instead of ``_temp``, consider ``closer_pos`` or something else similarly descriptive.